src/sinol_make/commands/inwer/__init__.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import os
+import multiprocessing as mp
+
+from sinol_make import util
+from sinol_make.commands.inwer.structs import TestResult, InwerExecution, VerificationResult, TableData
+from sinol_make.helpers import package_util, compile
+from sinol_make.helpers.parsers import add_compilation_arguments
+from sinol_make.interfaces.BaseCommand import BaseCommand
+from sinol_make.commands.inwer import inwer_util
+
+
+class Command(BaseCommand):
+    """
+    Class for "inwer" command.
+    """
+
+    def get_name(self):
+        return "inwer"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Verify if input files are correct',
+            description='Verify if input files are correct'
+        )
+
+        parser.add_argument('inwer_path', type=str, nargs='?',
+                            help='path to inwer file')
+        parser.add_argument('--tests', type=str, nargs='+',
+                            help='test to verify, for example in/abc{0,1}*')
```

Comment:
Can it be a regex? Seems important enough to mention it.

Reply:
The parameters are parsed by shell and not the program, so it can be anything that the shell allows